app/src/marketplace/components/Modal/ChooseAccessPeriodDialog/index.jsx

```diff
@@ -7,7 +7,7 @@ import { Translate, I18n } from 'react-redux-i18n'
 import Text from '$ui/Text'
 import useWeb3Status from '$shared/hooks/useWeb3Status'
 import LoadingIndicator from '$userpages/components/LoadingIndicator'
-import SelectField from '$mp/components/SelectField'
+import SelectField from '$shared/components/SelectField'
```

Comment:
The reason why the `SelectField` was under Marketplace is that they were styled according to editor with larger field height. Now the other components are gone and live under the `shared/Ui` folder. There is a "spacious" theme that gets used for example with text fields. There is already a select field under `Ui` but maybe this could joined with that. Most likely better to create another issue about it.

Reply:
Ah, must have missed that. I'll revert that and make a cleanup ticket for it.